dashboard/app/models/levels/level.rb

```diff
@@ -785,7 +785,7 @@ def summarize_for_lesson_show(can_view_teacher_markdown)
   # These properties are usually just the serialized properties for
   # the level, which usually include levelData.  If this level is a
   # StandaloneVideo then we put its properties into levelData.
-  def summarize_for_lab2_properties
+  def summarize_for_lab2_properties(script_level)
```

Comment:
Will use `script`. When I removed and called `super` I got an error.

Reply:
Did it not work if you included parentheses (ie, `super()`)? Seems like that should work, but yeah fine to leave as-is if not.

https://www.rubyguides.com/2018/09/ruby-super-keyword/

Reply:
I did try with and without parentheses 🤷‍♀️ but I'll try again!

Reply:
Looking at it again, this is the parent class and `Blockly` is the derived class so I don't think calling `super` here makes sense?
I do call `super` in `blockly.rb` [here](https://github.com/code-dot-org/code-dot-org/blob/c79d8d74f46e30f6549a11c4f5f06fb888659ae8/dashboard/app/models/levels/blockly.rb#L92).

Reply:
Oh I'm sorry for the confusion, I meant calling `super()` in `blockly.rb`, which I think would allow you to remove the parameter here.

Reply:
edit: I think I misread here, looks like this function gets called for all level types, not just Blockly levels (is that right?) A little confusing because the param only gets used in Blockly levels, but it does seem like you might have to keep this here.

Reply:
>edit: I think I misread here, looks like this function gets called for all level types, not just Blockly levels (is that right?) 

Yes - that's correct. An example of a level type that is lab2 but not Blockly is `aichat` which I confirmed calls on `summarize_for_lab2_properties` from `level.rb` but not `blockly.rb`. 

> A little confusing because the param only gets used in Blockly levels, but it does seem like you might have to keep this here.
  
👍   It looks like [this PR ](https://github.com/code-dot-org/code-dot-org/pull/24930) introduced `localized_blockly_level_options` which caches localization-specific option per-level, script, and locale. I just realized we don't translate blocks in non-Blockly labs like App Lab! Thanks for your questions which got me to look at the code more closely.

Reply:
Related discussion: https://github.com/code-dot-org/code-dot-org/pull/55457#discussion_r1427417493